Model/Sync/Reset/Orders.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Yotpo\Core\Model\Sync\Reset;
+
+class Orders extends Main
+{
+    const ORDERS_SYNC_TABLE = 'yotpo_orders_sync';
+    const ORDERS_TABLE = 'sales_order';
+    const ORDERS_DATA_LIMIT = 300;
+    const CRONJOB_CODES = ['yotpo_cron_core_orders_sync'];
+
+    /**
+     * @param int $storeId
+     * @return void
+     */
+    public function resetSync($storeId)
+    {
+        $this->setStoreId($storeId);
+        $this->setCronJobCodes(self::CRONJOB_CODES);
+        parent::resetSync($storeId);
+        $this->processResetSync();
+    }
+
+    /**
+     * @param int $offset
+     * @return void
+     */
+    public function processResetSync($offset = 0)
```

Comment:
processResetSync isn't right, because it is being called from `resetSync`. `resetSync` also processes. It should be have more revealing name.
```suggestion
    public function clearSyncTracks($offset = 0)
```

Reply:
This is done. Will update the PR soon